### PR TITLE
[CLOUD-3119] txrecovery module to run with python3

### PR DESCRIPTION
**Pull request by ochaloup**
https://issues.jboss.org/browse/CLOUD-3119

adding module for txrecovery which provides python3 version of the script which query OpenShift API. The module which is needed to be added to dependencies was renamed to `os-eap-txnrecovery.run` with version either `python2` or `python3`.
The dependency for the `os-eap-txrecovery` was removed from modules in the `eap-jboss-modules` and is expected it will be added to the `image.yaml` in the `jboss-eap-*-openshift-image` repositories.

It will be similar like
https://github.com/ochaloup/jboss-eap-7-openshift-image/commit/19b61473e286762a5520aa5824941eab764c431d

/CC @luck3y , would you be so kind and check if this is fine?

### CODE REVIEW

File: os-eap-txnrecovery/run/python2/module.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+schema_version: 1
+name: os-eap-txnrecovery.run
+version: 'python2'
+description: Query OpenShift API running with Python 2.
+execute:
+- script: install_as_root.sh
+packages:
+  install:
+    - python-enum34
```
Comment (luck3y) [inline os-eap-txnrecovery/run/python2/module.yaml]: This one, I don't think exists in RHEL 8 / UBI, but it appears to have been replaced by a py3 default. See here:

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html-single/8.0_beta_release_notes/index#package_replacements

(see python-enum34)

Comment (luck3y) [reply os-eap-txnrecovery/run/python2/module.yaml]: Yep, sorry my mistake, I misread the diff. This is just the python 2 version.

**Comment by luck3y** [comment]
@ochaloup Are you aware of any other products using this module as is other than EAP? Can you notify the CE list about this maybe? I think that should be sufficient. 

Already existing images will be build from stable branches / tags here, so nothing will change for people until they upgrade.

**Comment by luck3y** [comment]
@ochaloup Also, as a follow up to this, we'll need a CLOUD ticket to move the various images over into using it.

File: os-eap-txnrecovery/run/python2/module.yaml (hunk 1 above)
Comment (ochaloup) [reply os-eap-txnrecovery/run/python2/module.yaml]: yes, the package is needed only for the python2 where the enum is not available as I think from my limited knowledge of python

**Comment by ochaloup** [comment]
@luck3y:

> @ochaloup Are you aware of any other products using this module as is other than EAP? Can you notify the CE list about this maybe? I think that should be sufficient.

I think there is no other product using this. From what I know the Camel/Fuse created their own solution, similarly for Snowdrop/Spring. If the product is deployed on the JBoss EAP then lays on top of the EAP image and then it uses this solution automatically.
CE list is the cloud-enablement@redhat.com, correct?

Should I create CLOUD jira when this is merged or just now even the changes are not merged? Should I create one jira for all the image repositories or separate jiras should be created per repository? Am I right that the affected repositories are https://github.com/ochaloup/jboss-eap-7-openshift-image and https://github.com/ochaloup/jboss-eap-6-openshift-image while the change is needed to be done in all branches?

**Comment by luck3y** [comment]
@ochaloup Can create the issue now and just link it as related to this one. You can either assign it to yourself or me, up to you.

Thanks for doing this change Ondra! I'll build a test image for JDK 11 / UBI later today that has the change in it.

**Comment by ochaloup** [comment]
@luck3y I created https://issues.jboss.org/browse/CLOUD-3141 while I assigned that to you. But I'm fine to make those changes. I'm just a bit unsure what are  branches to work with. Or I meanI think all are for python2 currently, right? But I don't don't then where is the one for UBI.
If you give me a hint, feel free to assign that to me and I will do the work next week.

**Comment by luck3y** [comment]
@ochaloup Yeah, I can handle the UBI one, that's still a work-in-progress. If you want to, feel free to add it to branches eap64-dev, eap71-dev, eap72-dev and eap-cd-dev, that would be super helpful :)

I think they'll all remain python2 for the time being, with CD17 probably moving to py3 / UBI.

**Pull request merged**